src/DropdownToggle.js

```diff
@@ -82,4 +82,5 @@ class DropdownToggle extends React.Component {
   }
 }
 
+// Needs to be a class FTM, because it needs to accept a ref that can be used with findDOMNode
```

Comment:
why wouldn't a forwardRef into a functional component work?

Reply:
somewhere the ref doesn't get passed through, so we are actually relying on findDOMNode. I'll fix it up tho after i prune react-overlays and popper gets some hooks